internal/hostport/hostport.go

```diff
@@ -0,0 +1,75 @@
+package hostport
+
+import (
+	"net"
+
+	"github.com/cri-o/cri-o/internal/lib/sandbox"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"k8s.io/kubernetes/pkg/kubelet/dockershim/network/hostport"
```

Comment:
maybe we should stop vendoring and having it here?
cc: @danwinship

Reply:
I'm in favor of that.

Reply:
Just wondering how… @mrunalp would it be fine if we copy the manager sources into CRI-O and refactor / add testing?